src/streams/register_nack.rs

```diff
@@ -97,6 +98,20 @@ impl NackRegister {
         n
     }
 
+    pub fn accepts(&self, seq: SeqNo) -> bool {
+        let Some(active) = self.active.clone() else {
+            // if we don't have initialized, we do want the first packet.
+            return true;
+        };
+
+        // behind the window
+        if seq < active.start {
+            return false;
+        }
+
+        !self.packet(seq).received || seq > active.end
```

Comment:
how long is this window for active.end? I.e. if we had a 2 second burst loss would this seq number be outside the active window and we would drop it?

Reply:
It's not a fixed time, but a fixed number of packets. We call it `MAX_MISORDER`, if packets are arriving in sequence, the window shrinks. If they come out of order, we grow up to `MAX_MISORDER`.

Reply:
ok, so if we have:

SeqNo: 1,2,3,4 <crazy burst loss> 200, 201, 202, 203

the 200 packet will fit in the windows still and be inserted

Reply:
```rust
    #[test]
    fn nack_jump_larger_than_misorder() {
        let mut reg = NackRegister::new(None);
        for i in &[1, 2, 3, 4, 200, 201, 202, 203] {
            reg.update((*i).into());
        }

        assert_eq!(Some(103.into()..203.into()), reg.active)
    }
```

`MAX_MISORDER` is 100. So the active window goes from 103-203. I.e. we accept any packet as far back as 103. If you at this state run a nack report, you get 103-199 reported as missing. Sequence numbers 5-102 are silently forgotten.